views/event-form.hbs

```diff
@@ -0,0 +1,190 @@
+{{#if edit}}
+  <form class="add-form" action="/{{lang}}/edit-event/{{event._id}}" method="post">
+    <h1>{{localText.edit}}</h1>
+    <input type="hidden" name="_method" value="put">
+{{else}}
+  <form class="add-form" action="/{{lang}}/add-event" method="post">
+    <h1>{{localText.add}}</h1>
+    <input type="hidden" name="_method" value="post">
+{{/if}}
+
+<span>{{localText.formInfo}}</span>
+<fieldset name="en" class="langFieldset">
+  <legend>{{localText.english}}</legend>
+  <label>{{localText.eventsName}}
+    {{#if event.en.name}}
+      <input type="text" name="name_en" value="{{event.en.name}}">
+    {{else}}
+      <input type="text" name="name_en">
+    {{/if}}
+  </label>
+
+  <label>{{localText.description}}
+    {{#if event.en.description}}
+      <input type="text" name="description_en" value="{{event.en.description}}">
+    {{else}}
+      <input type="text" name="description_en">
+    {{/if}}
+  </label>
+</fieldset>
+
+<fieldset name="ar" class="langFieldset">
+  <legend>{{localText.arabic}}</legend>
+  <label>{{localText.eventsName}}
+    {{#if event.ar.name}}
+      <input type="text" name="name_ar" value="{{event.ar.name}}">
+    {{else}}
+      <input type="text" name="name_ar">
+    {{/if}}
+  </label>
+
+  <label>{{localText.description}}
+    {{#if event.ar.description}}
+      <input type="text" name="description_ar" value="{{event.ar.description}}">
+    {{else}}
+      <input type="text" name="description_ar">
+    {{/if}}
+  </label>
+</fieldset>
+
+
+<label><span style="color:red;">*</span>
```

Comment:
also we're not doing inline styling anywhere else so let's not do it here? The * doesn't have to red at all tbh so it doesn't need to be in the span element at all I guess